lib/modules/manager/maven-wrapper/extract.ts

```diff
@@ -15,10 +15,16 @@ const WRAPPER_URL_REGEX = regEx(
   '^(?:wrapperUrl\\s*=\\s*)(?<url>\\S*-(?<version>\\d+\\.\\d+(?:\\.\\d+)?(?:-\\w+)*)(?:.jar))',
 );
 
+const WRAPPER_VERSION_REGEX = regEx(/wrapperVersion=(?<version>\d+(\.\d+)*)/);
```

Comment:
are you sure those versions are allowed:
- `3`
- `3.8`
- `3.8.1.2`
i think this is better:
```suggestion
const WRAPPER_VERSION_REGEX = regEx(/wrapperVersion=(?<version>\\d+\\.\\d+(?:\\.\\d+)?(?:-\\w+)*)/);
```
use the same regex like on the url and support prereleases

Reply:
I tought my regex did support it, but fine with yours as wel!